ExamPortal/Models/question.js

```diff
@@ -0,0 +1,28 @@
+const mongoose  = require('mongoose')
+
+const Schema = mongoose.Schema
+
+const questionSchema = new Schema({
+	qHeading: String,
+	qText: String,
+	qNo: Number,
```

Comment:
What exactly is this? Will it decide the sequence of the question. If so, will user enter the sequence of the question?